tests/test_SimSCPIFormatting.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+import yaml
+import os
+from basil.dut import Dut
+
+k2410def_yaml = os.path.join(os.path.dirname(__file__), "formatting.yaml")
+
+cnfg_yaml = f'''
+transfer_layer:
+  - name     : Visa
+    type     : Visa
+    init     :
+        resource_name : ASRL1::INSTR
+        read_termination : "\\n"
+        write_termination : "\\r\\n"
+        backend : "{k2410def_yaml}@sim"
+
+hw_drivers:
+  - name      : Sourcemeter
+    type      : scpi
+    interface : Visa
+    init      :
+        device : Keithley 2410
```

Comment:
Would there be a way to automatically enable formating by setting something here?

Reply:
One could enable it from the device configuration, yes. Maybe by defining something like `enable_formatting: true`

Reply:
I think it is OK to go as is. You can add this `enable_formatting: true` for your own convenience.